.golangci.yml

```diff
@@ -11,10 +11,22 @@ linters-settings:
   govet:
     # report about shadowed variables
     check-shadowing: true
+    enable-all: true
+    disable:
+      - fieldalignment
+
+  revive:
+    rules:
+      - name: package-comments
+        disabled: true
```

Comment:
I took the `golangci.yml` from `atomic-swap` and made 2 changes:
* Disabled the package-comments requirement (code at this comment)
* Removed  the tagliatelle checks for camel cased JSON tags